doc/source/reference/testing.rst

```diff
@@ -58,8 +58,6 @@ Exceptions and warnings
    errors.PossiblePrecisionLoss
    errors.PyperclipException
    errors.PyperclipWindowsException
-   errors.SettingWithCopyError
```

Comment:
If you have another PR with the whatsnew note, please make sure that the removal of these are mentioned

Reply:
I'll create a separate PR with a whatsnew